HiParsely/HiParsely/ViewController.m

```diff
@@ -69,15 +69,9 @@ -(void)updateQueueStatusLabel{
 -(void)toggleSimulatedConnection{
     if(hasConnection){
         hasConnection = NO;
-#ifdef PARSELY_DEBUG
```

Comment:
These haven't been used in ages, removing them should be fine. The original purpose was to fake a connection being unavailable for testing purposes.